src/elements/content-sidebar/activity-feed/common/activity-message/ActivityMessage.scss

```diff
@@ -23,6 +23,18 @@
     color: $bdl-box-blue;
 }
 
+.bcs-ActivityMessage-edited {
+    color: $bdl-gray-62;
+
+    &::before {
+        content: ' (';
```

Comment:
Brackets and parentheses are grammatical/linguistic and depend on the locale. As such, they should be part of the text for translation. ie. "(edited)" rather than "(" + translate("edited") + ")"

Reply:
Also, some languages do not use space between words or phrases, so " (" is not appropriate. Put the space into the string to translate as well, so that the translators can do the right thing with each space. ie. " (edited)"

Reply:
Now that you mentioned it it's obvious. Thanks for explenation!